packages/cloud_functions/cloud_functions/lib/src/firebase_functions.dart

```diff
@@ -96,6 +96,44 @@ class FirebaseFunctions extends FirebasePluginPlatform {
         delegate.httpsCallableWithUri(_origin, uri, options));
   }
 
+  /// A reference to the streaming Callable HTTPS trigger with the given name.
+  ///
+  /// Should be the name of the Callable function in Firebase that supports streaming.
```

Comment:
Here we should copy the entire doc from the java implementation (and adapt it if needed)